scss/_buttons.scss

```diff
@@ -1,23 +1,23 @@
 .btn {
-  border-width: $border-width;
   font-weight: $font-weight-semibold;
-  border-radius: $btn-border-radius;
   white-space: nowrap;
+  border-width: $border-width;
+  @include border-radius($btn-border-radius);
 
-  @include hover-focus {
+  @include hover-focus () {
```

Comment:
Ooohh interesting This could interest me but I don't know if it should be done in this PR when using something like `<button type="button"><i class="material-icons>edit</i></button>` There is no hover effect (color change) Wheras with an `<a class="tooltip-link"><i class="material-icons>edit</i></a>` There is one, it would be great to have the same effect on pure buttons (even if it's linked to an additional class to avoid messing with the `btn` class)

We discussed about this @Progi1984 here https://github.com/PrestaShop/PrestaShop/pull/16150/files#r353677243 and I have the same need on a CLDR PR

Reply:
Yes It's not linked to that PR at all, create an issue or comment on the Button improvement one :)

Reply:
Well actually since the tag have been removed and there are only classes, we can then use `tooltip-link` on a button (even though it is not semanticaly correct, so maybe one day we should add something for the btn class, or an alternative btn-class)